config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   resources :users, only: [:new, :create, :show, :edit, :update, :destroy]
   resources :budgets, only: [:new]
+  resources :sessions, only: [:new, :create, :destroy]
+  post "/sessions/new", to: "sessions#create"
```

Comment:
I still don't think line 5 is needed if it is defined on line 4

Reply:
I keep getting a no route matches post sessions/new

Reply:
We don't usually use the `.../new` path for post actions.
We usually use the plural `sessions_path` for the post.

When removing line 5, you will need to add a `url` param to the new session form (line 3). I.e.
`<%= simple_form_for :session, url: sessions_path do |f| %>`
Because sessions are not ActiveRecord resource, simple_form needs a little help.

Reply:
Aah, yes, will take a look, thank you